components/common/ConnectWallet/WalletAssetMenu.vue

```diff
@@ -78,11 +78,6 @@ const menus = ref<{ label: string; to: string; vm: ChainVM[] }[]>([
     to: `/${urlPrefix.value}/teleport`,
     vm: ['SUB'],
   },
-  {
-    label: 'Migrate',
-    to: '/migrate',
-    vm: ['SUB'],
-  },
```

Comment:
would keep migrator. 
cc @damskyftw ?

Reply:
Really? If we keep the migrate section, we have to maintain the indexer for rmrk/ksm and keep it runing.

Reply:
Would just hide the route